### PR TITLE
Monoscopic controls

### PR DESCRIPTION
**Pull request by rapka**
This is a first pass at updating the controls to work in 2D. I'm sure that the constant checks for `IsMonoscopicMode()` can be refactored away, but this is my first PR and that was the most intuitive way to avoid changing the existing mouse+keyboard controls that currently seem to apply to the spectator camera view.

Controls:
WASD+Mouse : Movement
Spacebar: Move camera up
Left control: Move camera down
Scroll wheel: Increase/decrease movement speed
F/R: decrease/increase cursor distance from camera
Shift: toggle drawing in 2D

### CODE REVIEW

File: Assets/Editor/BuildTiltBrush.cs [hunk 1]
```diff
@@ -309,7 +309,7 @@ public static TiltBuildOptions GetGuiOptions()
     }
 
     // Menu items
-    [MenuItem("Tilt/Build/Do Build...", false, 2)]
+    [MenuItem("Tilt/Build/Do Build... #&b", false, 2)]
```
Comment (rapka) [inline Assets/Editor/BuildTiltBrush.cs:312]: This adds shift+alt+b as a shortcut.

File: tiltasaurus.json [hunk 2]
```diff
@@ -1,274 +0,0 @@
-{
```
Comment (rapka) [inline tiltasaurus.json:1]: As far as I could tell, this file is completely unused.

**Comment by mikeage** [comment]
I'm not sure why, but even thought it ran the CI checks on commit cd7eb8c, it didn't run them on the latest. I've triggered this manually.

**Comment by mikeage** [comment]
@andybak / @billyquith / @mikeskydev any requests or should we merge this? From my POV, the controls are more intuitive now.

**Review by andybak — COMMENTED** [review]
This does affect a fair few parts of the code and probably needs some decent testing in real-world use. It will be easier for me to test once it's merged and we can (hopefully) persaude some other people to use the beta as their daily driver. 

Small point - I'd probably have preferred not renaming MonoCameraControlScript as it might be more hassle to merge in other people's stuff. Plus we lose the git history (having said that - there's not much in this file). Whilst the new name is slightly better, it's not really worth the pain for the gain. But - it's done now and I am splitting hairs to some degree.

**Comment by andybak** [comment]
> 
> 
> This does affect a fair few parts of the code and probably needs some decent testing in real-world use. It will be easier for me to test once it's merged and we can (hopefully) persaude some other people to use the beta as their daily driver.
> 
> Small point - I'd probably have preferred not renaming MonoCameraControlScript as it might be more hassle to merge in other people's stuff. Plus we lose the git history (having said that - there's not much in this file). Whilst the new name is slightly better, it's not really worth the pain for the gain. But - it's done now and I am splitting hairs to some degree.

EDIT - So I've just discovered reviews aren't discussions and I only get one comment... Hmmmmm. In summary - I want to do a bit more testing before approving.

**Comment by billyquith** [comment]
If people can test the PR, [GH command-line tool](https://cli.github.com/) makes this easy.

**Comment by andybak** [comment]
> If people can test the PR, [GH command-line tool](https://cli.github.com/) makes this easy.

Not sure I understand? The kind of testers I was expecting to come our way probably want links to apks and exes - and probably want nothing at all to do with git.

**Comment by mikeskydev** [comment]
I gave this a quick test, and while it worked for the most part, I kept getting into a confusing loop where it seemed to save and load the last sketch I had rapidly, and eventually froze the application.

**Comment by mikeage** [comment]
The Quest builds had failed because of #171 ; while updating that, I also ran pre-commit once manually

**Comment by andybak** [comment]
There's still some major flaws with this - but there's some major flaws with the existing monoscopic mode as well.

This would be really useful as a separate mode for viewing sketches on a desktop PC but I'm not sure I want to make hard-to-reverse changes to the current monoscopic mode until this is a fully fledged replacement. 

So I guess I'm saying "let's close this PR and revisit it when either we're sure it's at least as good as the current mode, or we can refactor it to be an alternative to the current mode"

**Pull request closed without merging**